feat: Crews

kyranet opened the pull request:
WIP

kyranet committed:
Añadidos los comandos y sistemas para las tripulaciones

StartForKiller reviewed (approved):
Aparentemente está todo correcto, aprobado.

Commit:
Consolidados comandos de tripulaciones en el comando tripulacion y sus subcomandos
Agregadas opciones para ver una lista de tripulaciones, visualizacion y edicion de tripulaciones, opcion de abandonar para tripulantes y comandos de moderacion y borrado para el administrador

Commit:
Fixed broken code indentation

kyranet committed:
Update src/commands/Tripulacion/tripulacion.js

Co-Authored-By: GGijon <guillermogijon@gmail.com>

Commit:
Resueltas advertencias de eslint

Commit:
Multiples cambios solicitados por Kyra

Commit:
Mas cambios solicitados por Kyra

kyranet committed:
Merge pull request #20 from GGijon/crews

Tripulacion
